cross/binutils/Makefile

```diff
@@ -7,6 +7,10 @@ include ../../mk/spksrc.main-depends.mk
 
 ifeq ($(findstring $(ARCH),$(OLD_PPC_ARCHS)),$(ARCH))
 DEPENDS = cross/binutils-2.32
+else ifeq ($(call version_lt, $(TCVERSION), 7),1) 
```

Comment:
Would probably be simpler to use gcc version?

Reply:
@th0ma7 you're right - and with additional CXXFLAG much more toolchains are supported :smile: